src/server/server-middlewares.js

```diff
@@ -39,4 +41,8 @@ export default (app: Router, env: any) => {
     log.error(error)
     res.status(400).json({ message: error.message })
   })
+
+  // register & start cron tasks
```

Comment:
forgot that DisposeEnrollmentsTask returns factory 
so you should do:

```
import createDisposeEnrollmentsTask from './verification/cron/DisposeEnrollmentsTask'


const disposeEnrollmentsTask = createDisposeEnrollmentsTask(UserDBPrivate)

CronTasksRunner.registerTask(disposeEnrollmentsTask)
CronTasksRunner.startTasks()
```

Reply:
Also we should handle express app termination (or process exit / SIGINT / SIGTERM event) and call CronTasksRunner.stopTasks()

Reply:
@serdiukov-o-nordwhale 
the changes with DisposeEnrollmentsTask Stas already did.
I added changes for process exit / SIGINT / SIGTERM events here - https://github.com/GoodDollar/GoodServer/pull/177